### PR TITLE
Code sample - RSI DataLoad Mode

### PR DESCRIPTION
**Pull request by juarezjuniorgithub**
Signed-off-by: Juarez Barbosa juarez.barbosa@oracle.com
@Kuassim @jeandelavarene please review and approve. Thanks!

### CODE REVIEW

File: java/rsi-dataload-mode/pom.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+		<dependency>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>rsi</artifactId>
+			<version>23.3.0.23.09</version>
+		</dependency>
+
+		<dependency>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>ojdbc11</artifactId>
+			<version>23.3.0.23.09</version>
```
Comment (jeandelavarene) [inline java/rsi-dataload-mode/pom.xml:34]: Please use 23.4

Comment (juarezjuniorgithub) [reply java/rsi-dataload-mode/pom.xml]: Done

Comment (jeandelavarene) [reply java/rsi-dataload-mode/pom.xml]: Please use [23.4.0.24.05](https://mvnrepository.com/artifact/com.oracle.database.jdbc/ojdbc11/23.4.0.24.05)

**Comment by juarezjuniorgithub** [comment]
@Kuassim @jeandelavarene the adjustments were performed as requested - fixed indentation.
Please review/approve. Thanks.

**Pull request closed without merging**